mantidproject/mantid
# MA now has a checkbox for autoscale

**Description of work.**
Wanted to add a checkbox for MA and FDA.
Unit tests will be added in #29321


**To test:**
Open MA/FDA
Load data
Check if the autoscale works (e.g. change data, rebin, change groups/pairs)
Do not use fitting it has a known bug

Open Elemental analysis
Load data
Create a plot
The autoscale should still be a button


Fixes #29065. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
